radicle-httpd/src/api/v1/sessions.rs

```diff
@@ -131,5 +153,8 @@ mod routes {
         .await;
 
         assert_eq!(response.status(), StatusCode::OK);
+
+        let response = get(&app, format!("/sessions/{}", session_info.session_id)).await;
+        assert_eq!(response.status(), StatusCode::OK);
```

Comment:
An improvement on the test would be to GET before the PUT, check that it's unauthorized, then GET after the PUT, and check that it's authoriezed.

Reply:
Ok yeah makes sense done in 1150daa4e04ef0195d3e478beb3840fcf5b014fe